docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_home_page.yml

```diff
@@ -0,0 +1,71 @@
+# Migrates all revisions and all revision translations.
+id: yukon_migrate_home_page
+label: Yukon Migrate Home Page Node
+audit: true
+migration_tags:
+  - Drupal 7
+  - node
+  - Home Page
+migration_group: legacy
+source:
+  plugin: d7_node
+  node_type: homepage
+  constants:
+    default_text_format: full_html
+process:
+  langcode:
+    plugin: default_value
+    source: language
+    default_value: "en"
+  title: title
+  uid: node_uid
+  status:
+    plugin: skip_on_value
+    method: row
+    not_equals: true
+    value: 1
+    source: status
+  created: created
+  changed: timestamp
+  promote: promote
+  sticky: sticky
+
+  metatags/0/description: field_page_description/0/value
```

Comment:
Create a separate field as it was in D7 and migrate value into it. Matatags should be configured to use tat field token. So do not migrate anything directly into matatags please

Reply:
This should now be resolved.